### DISCUSSION TITLE
Полезное

### DISCUSSION THREAD

**Discussion by Victor-P-21** [Victor-P-21/TestFight#7] [Show and tell]
Здесь я буду выкладывать полезные ссылки если посчитаю необходимым

**Comment by Victor-P-21** [comment]
Немного о названиях переменных и функций:
https://habr.com/ru/post/477722/

https://ru.wikipedia.org/wiki/%D0%A1%D0%BE%D0%B3%D0%BB%D0%B0%D1%88%D0%B5%D0%BD%D0%B8%D1%8F_%D0%BE%D0%B1_%D0%B8%D0%BC%D0%B5%D0%BD%D0%B0%D1%85_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)#Python_%D0%B8_Ruby

https://ru.wikipedia.org/wiki/CamelCase

**Reply by GStEugene** [reply]
Постараюсь писать следуя данной условности

**Reply by Victor-P-21** [reply]
Ну справедливости ради я сам часто переключаюсь с кэмел кейса на снейк или оба сразу, да и названия от балды зачастую ставлю или от важности к примеру переменной, но общие стандарты примерно такие... и не забудь изучить почему все щитают опасными глобальные переменные
